tests/pip_install/whl_library/generate_build_bazel_tests.bzl

```diff
@@ -219,6 +249,74 @@ py_binary(
 
 _tests.append(_test_with_entry_points)
 
+def _test_group_member(env):
+    want = """\
+load("@rules_python//python:defs.bzl", "py_library", "py_binary")
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "dist_info",
+    srcs = glob(["site-packages/*.dist-info/**"], allow_empty = True),
+)
+
+filegroup(
+    name = "data",
+    srcs = glob(["data/**"], allow_empty = True),
+)
+
+filegroup(
+    name = "_whl",
+    srcs = ["foo.whl"],
+    data = ["@pypi_bar_baz//:whl"],
+)
+
+py_library(
+    name = "_pkg",
+    srcs = glob(
+        ["site-packages/**/*.py"],
+        exclude=[],
+        # Empty sources are allowed to support wheels that don't have any
+        # pure-Python code, e.g. pymssql, which is written in Cython.
+        allow_empty = True,
+    ),
+    data = [] + glob(
+        ["site-packages/**/*"],
+        exclude=["**/* *", "**/*.py", "**/*.pyc", "**/*.pyc.*", "**/*.dist-info/RECORD"],
+    ),
+    # This makes this directory a top-level in the python import
+    # search path for anything that depends on this.
+    imports = ["site-packages"],
+    deps = ["@pypi_bar_baz//:pkg"],
+    tags = [],
+)
+
+alias(
+   name = "pkg",
+   actual = "@pypi__groups//:qux_pkg",
+)
+
+alias(
+   name = "whl",
+   actual = "@pypi__groups//:qux_whl",
+)
+"""
+    actual = generate_whl_library_build_bazel(
+        repo_prefix = "pypi_",
+        whl_name = "foo.whl",
+        dependencies = ["foo", "bar-baz", "qux"],
```

Comment:
The `whl_installer` is going to remove `self` from the dependency list, so `"foo"` is likely to never be present in the `dependencies` list here.

Reply:
Yeah this is just copypasta from the previous test https://github.com/bazelbuild/rules_python/pull/1166/files#diff-864e4b8f44994e69a2522a561ca85c7721aa1bd869ef9223daf9996d921cdb24R242. Happy to clean both or just this instance up.